config/initializers/secure_headers.rb

```diff
@@ -0,0 +1,8 @@
+SecureHeaders::Configuration.configure do |config|
+  # rubocop:disable Lint/PercentStringArray
+  config.csp = {
```

Comment:
🚧 not a blocker
As a matter of interest, why are we disabling rubocop rather than just changing the arrays to 
```ruby
%w[none]
```
?

Reply:
I wasn't too happy about that either. The gem needs the text to be in single quotes - if I run it without the quotes it throws: 

```default_src contains an invalid keyword source (none). This value must be single quoted. (SecureHeaders::ContentSecurityPolicyConfigError)```

The only way I could get it to play nicely with rubocop was by disabling the cop.

Reply:
I suppose the only other way would be to drop the array modifier completely and just use
```ruby
["'none'"]
```
But I'm not sure it's any better! 😃